src/gui/resources/translations/de.ts

```diff
@@ -0,0 +1,2749 @@
+    </message>
+    <message>
+        <location filename="../../kperfwidget.ui" line="815"/>
+        <source>Note: macro recording not yet implemented. Indicator will always display &quot;off&quot;.</source>
+        <translation>Hinweis: Makro-Aufzeichnungen sind noch nicht implementiert. Der Indikator zeigt immer &quot;aus&quot; an.</translation>
+    </message>
+    <message>
+        <location filename="../../kperfwidget.ui" line="916"/>
+        <source>Unknown</source>
+        <translation>Unbekannt</translation>
+    </message>
+    <message>
+        <location filename="../../kperfwidget.ui" line="949"/>
+        <source>K95 buttons</source>
+        <translation>K95-Tasten</translation>
+    </message>
+    <message>
+        <location filename="../../kperfwidget.ui" line="962"/>
+        <source>Note: many desktop environments do not use scroll lock. The indicator may not turn on even when enabled.</source>
+        <translation>Hinweis: Viele Desktopumgebungen benutzen Scroll lock (Rollen) nicht. Der Indikator leuchtet möglicherweise nicht, auch wenn er aktiviert ist.</translation>
+    </message>
+    <message>
+        <location filename="../../kperfwidget.ui" line="1022"/>
+        <source>If the sound device does not support muting, or if mute status is unknown, this color will be displayed.</source>
+        <translation>Wenn das Audiogerät die Stummschaltung nicht unterstützt oder wenn der Stummschaltungsstatus unbekannt ist, wird diese Farbe angezeigt.</translation>
+    </message>
+    <message>
+        <location filename="../../kperfwidget.ui" line="1034"/>
+        <source>Indicator intensity:</source>
+        <translation>Indikator-Intensität:</translation>
+    </message>
+    <message>
+        <location filename="../../kperfwidget.cpp" line="227"/>
+        <source>Copy performance settings to:</source>
+        <translation>Leistungseinstellungen kopieren nach:</translation>
+    </message>
+</context>
+<context>
+    <name>KStatusNotifierItem</name>
+    <message>
+        <location filename="../../kstatusnotifier/kstatusnotifieritem.cpp" line="578"/>
+        <source>Quit</source>
+        <translation>Beenden</translation>
+    </message>
+</context>
+<context>
+    <name>KbAnimWidget</name>
+    <message>
+        <location filename="../../kbanimwidget.ui" line="48"/>
+        <source>No animations in this mode</source>
+        <translation>Keine Animationen in diesem Modus</translation>
+    </message>
+    <message>
+        <location filename="../../kbanimwidget.ui" line="176"/>
+        <source>Normal</source>
+        <translation>Normal</translation>
+    </message>
+    <message>
+        <location filename="../../kbanimwidget.ui" line="181"/>
+        <source>Add</source>
+        <translation>Hinzufügen</translation>
+    </message>
+    <message>
+        <location filename="../../kbanimwidget.ui" line="186"/>
+        <source>Subtract</source>
+        <translation>Subtrahieren</translation>
```

Comment:
I used _Hinzufügen, Subtrahieren..._ because it's used for the blend mode menu and drawing programs usually translate it as that.



<details>
  <summary>Procreate - Subtract, Subtrahieren</summary>

![Procreate_de_en](https://user-images.githubusercontent.com/30179385/128486722-c9fbefe7-2281-4830-a89c-f13742af1482.png)

</details>

<details>
  <summary>Procreate - Add, Hinzufügen</summary>

![Procreate-add_de_en](https://user-images.githubusercontent.com/30179385/128495449-dfb16b32-579d-4175-8beb-3bde3cdd7f35.png)


</details>

<details>
   <summary>In ckb-next</summary>

![ckb-next-blendmode_de_en](https://user-images.githubusercontent.com/30179385/128489577-a8443c82-2459-4a14-bc81-0988e44f2d49.png)


</details>

Reply:
@tatokis, that’s why he was confused. If you talk about calculations like `1+1` or `2-1`, you’ll use “Addieren” and “Subtrahieren”. “Hinzufügen” and “Entfernen” are more like add/remove an item from a list, so it’s weird why the person that came up with these terms in the first place would mix them.

Reply:
Here it is the mathematical meaning of add: it's how two animations get combined, so the results get added together. If the distinction here is between mathematically adding and adding to a list I think the mathematical meaning should win out.

Reply:
Yeah, so it looks like I phrased my response really poorly. I meant to say that given that it's mathematical calculations, it should probably be Addieren and Subtrahieren